patches/server/0978-add-getHighestMapId-method.patch

```diff
@@ -0,0 +1,37 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: chmodsayshello <chmodsayshello@hotmail.com>
+Date: Mon, 12 Jun 2023 18:56:48 +0200
+Subject: [PATCH] add getHighestMapId method
+
+
+diff --git a/src/main/java/net/minecraft/world/level/saveddata/maps/MapIndex.java b/src/main/java/net/minecraft/world/level/saveddata/maps/MapIndex.java
+index 9b2948b5150c8f039ca667a50765109721b93947..9f69db46239d3e8055d5793a6a9ba62d321b5e4f 100644
+--- a/src/main/java/net/minecraft/world/level/saveddata/maps/MapIndex.java
++++ b/src/main/java/net/minecraft/world/level/saveddata/maps/MapIndex.java
+@@ -40,4 +40,10 @@ public class MapIndex extends SavedData {
+         this.setDirty();
+         return i;
+     }
++
++    // Paper start - get value without writing
++    public int getCurrentValueForMap() {
++        return this.usedAuxIds.getInt("map");
```

Comment:
You could possibly use an [AT](https://github.com/PaperMC/Paper/blob/master/CONTRIBUTING.md#access-transformers) here instead of a new method.

Reply:
Nah it's fine for a single thing